src/LDNS.xs

```diff
@@ -1375,32 +1375,42 @@ packet_edns_version(obj,...)
 # -------------------
 # Get/set EDNS data
 # 
-# Beware, this code can only take a unique U32 parameter which means it 
-# is not a full implementation of EDNS data but it is enough for our 
+# This function will set EDNS RDATA in a packet when given a parameter,
+# otherwise it will return the entire EDNS RDATA of the packet (if any).
+#
+# Beware, when setting EDNS RDATA, this code can only take a unique U32 parameter
+# which means it is not a full implementation of EDNS data but it is enough for our
 # current purpose. It can only deal with option codes with OPTION-LENGTH=0
 # (see 6.1.2 section of RFC 6891) which means OPTION-DATA is always empty.
 #
-# returns: a bytes string
+# returns: a bytes string (or undef if no EDNS data is found)
```

Comment:
Does "EDNS data" refer to an non-empty OPT RDATA, i.e. EDNS option code and data?

Reply:
Yes, I think. This is returned from LDNS by this [function](https://www.nlnetlabs.nl/documentation/ldns/packet_8h.html#ad55346b7f33de49bf4f3e34fdaa412c6).

Reply:
There is other EDNS data so that should be clarified.